README.md

```diff
@@ -49,8 +49,8 @@ Pre-built pip packages support Ubuntu 18.04+, macOS 10.15+ and Windows 10+
 
 ```bash
 # Install
-pip install open3d       # or
-pip install open3d-cpu   # Smaller CPU only wheel on x86_64 Linux (v0.17+)
+pip install open3d[standard]       # or
+pip install open3d-cpu[standard]   # Smaller CPU only wheel on x86_64 Linux (v0.17+)
```

Comment:
Same, please add the behavior of `pip install open3d` and `pip install open3d-cpu`. Ideally, elaborate on the design choice and the expected usage of both options.